Lab3.ipynb

```diff
@@ -0,0 +1,366 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = np.vstack((stats.norm.rvs( scale = 1, size = 100000), stats.norm.rvs( scale = 1, size = 100000)))\n",
+    "a.shape\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "h = ax.hist2d(a[0,:],a[1,:],bins=100, density=True);\n",
+    "ax.set_aspect('equal', 'box')\n",
+    "plt.xlim([-3 , 3])\n",
+    "plt.ylim([-3 , 3])\n",
+    "plt.title(\"2D Histogram of positional uncertainty\", fontsize = 24)\n",
+    "plt.ylabel(\"$\\Delta$y arcseconds\", fontsize = 18)\n",
+    "plt.xlabel(\"$\\Delta$x arcseconds\", fontsize = 18)\n",
+    "plt.colorbar(h[3], ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 2.1**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The statistical question I have is what is the displacement or movement to see a 5-sigma detection from the background in arcseconds? The reason this is my statistical question is because an asteroid will have some true movement between the images. "
```

Comment:
I would mentions the analytic pdf that we are dealing with ( the Rayleigh you are using in the next section) yea I would also explain the math you do in part 3 here